src/theory/theory_engine.cpp

```diff
@@ -876,13 +876,8 @@ bool TheoryEngine::solve(TrustNode tliteral,
   // Note that ppAssert is called before ppRewrite.
   if (!isTheoryEnabled(tid) && tid != THEORY_SAT_SOLVER)
   {
-    stringstream ss;
-    ss << "The logic was specified as " << logicInfo().getLogicString()
-       << ", which doesn't include " << tid
-       << ", but got a theory atom for that theory." << std::endl
-       << "The atom:" << std::endl
-       << atom;
-    throw LogicException(ss.str());
+    // don't throw an exception yet
```

Comment:
where is this exception thrown now? It seems that it is supposed to be caught at ppStaticRewrite time. Is that test as complete as this one?